unit_tests/test_fuzz_logger_text.py

```diff
@@ -456,7 +456,7 @@ def test_several(self):
         self.virtual_file.seek(0)
         self.assertTrue(self.some_test_case_id in self.virtual_file.readline())
         self.assertTrue(self.some_test_step_msg in self.virtual_file.readline())
-        self.assertTrue(self.some_recv_data in self.virtual_file.readline())
+        self.assertTrue(str(len(self.some_recv_data)) in self.virtual_file.readline())
```

Comment:
@jtpereyda Why do the assertions for `log_recv()` and `log_send()` differ each other? 

Besides,  the assertions for log_recv() elsewhere seem inconsistent. 

1) using `fuzz_logger_text.DEFAULT_HEX_TO_STR(self.some_recv_data)`
https://github.com/jtpereyda/boofuzz/blob/0a78c15a781b9feea5321a0f350e3ad28ee0da68/unit_tests/test_fuzz_logger_text.py#L197-L212

2) using `self.some_recv_data` directly
https://github.com/jtpereyda/boofuzz/blob/0a78c15a781b9feea5321a0f350e3ad28ee0da68/unit_tests/test_fuzz_logger_text.py#L453-L455

Reply:
that's a great question, I will look into why that's like that

Reply:
According to the format of log, it seems both `self.assertTrue(self.some_recv_data in self.virtual_file.readline())` and `self.assertTrue(str(len(self.some_recv_data)) in self.virtual_file.readline())` would work.

Reply:
This would only work for Python2 though. Python3 expects a string type for the left operand of "a in b"
```python
TypeError: 'in <string>' requires string as left operand, not bytes
unit_tests/test_fuzz_logger_text.py:459: TypeError
```
In the current state the test passes, but I don't think it works as intended. Right now it only check if the `len(self.some_recv_data)` exists, when it actually should check if `self.some_recv_data` itself does.
So this is what I belive the line should look like (correct me if I'm wrong):
```python
self.assertTrue(str(self.some_recv_data) in self.virtual_file.readline())
```
@cq674350529 Good catch! I wouldn't have noticed as the test passes either way.
@aedrax I'll include this in the next PR on your branch or you can just change it yourself.

Reply:
Be aware that send/recv data can be truncated in some loggers... I don't _think_ this occurs in FuzzLoggerText though.

I may have loosened up a check after changing the format of the output. I'm fine with somebody making the check more strict as long as it makes sense and tests still pass.